src/App.js

```diff
@@ -338,7 +338,7 @@ export function Dashboard({
     return (
       <div>
         <h2 style={{marginBottom: '1em', display: 'flex', flexWrap: 'wrap'}}>
-          {capitalize(product)} {version} - Channel: {releaseInfo.channel}{' '}
```

Comment:
So you remove it for all channels eventually?

Reply:
I believe this would answer @sylvestre's comment on irc about "this info not being useful".